example/lib/bar_chart/samples/bar_chart_sample2.dart

```diff
@@ -160,6 +172,7 @@ class BarChartSample2State extends State<BarChartSample2> {
                     ),
                     barGroups: showingBarGroups,
                     gridData: FlGridData(show: false),
+                    extraLinesData: extraLinesData,
```

Comment:
You updated two samples to demonstrate the extraLinesData. They don't fit with the samples design.
You can also add another BarChartSample to Implement something like the below chart(taken from [this comment](https://github.com/imaNNeoFighT/fl_chart/issues/476#issuecomment-724771206)):
<img width="400" alt="image" src="https://user-images.githubusercontent.com/7009300/213139815-b17a9cab-7a26-4a12-9045-fb2461fc92b4.png">